- type: fixed
  marker_text: kitchen
  marker_type: arrow
  marker_position:
    x: 0.25
    y: -1.62
  marker_scale:
    x: 0.025
    y: 0.025
    z: 0.1
  marker_color:
    r: 0
    g: 0
    b: 0
  marker_orientation:
    theta: 2.75
- type: fixed
  marker_text: home
  marker_type: arrow
  marker_position:
    x: 0
    y: 0
  marker_orientation:
    theta: 0
  marker_scale:
    x: 0.1
    y: 0.025
    z: 0.005
  marker_color:
    r: 0
    g: 0
    b: 0
- type: fixed
  marker_text: entrance
  marker_type: arrow
  marker_position:
    x: -0.3
    y: -0.1
  marker_orientation:
    theta: 2.9
  marker_scale:
    x: 0.1
    y: 0.025
    z: 0.005
  marker_color:
    r: 0
    g: 0
    b: 0
- type: fixed
  marker_text: computer
  marker_type: arrow
  marker_position:
    x: 2.0
    y: 0.1
  marker_orientation:
    theta: -0.57
  marker_scale:
    x: 0.1
    y: 0.025
    z: 0.005
  marker_color:
    r: 0.25
    g: 0.1
    b: 0.3